config/config.go

```diff
@@ -208,9 +208,15 @@ func validateReferencesExist(c *configpb.Configuration) error {
 	return mErr
 }
 
-// validateName validates an entity name is non-empty and contains no prefix that overlaps with a
-// TestGrid file prefix, post-normalization.
+// TODO(michelle192837): Remove '/' and '–' from this regex.
+var nameRegex = regexp.MustCompile("^[a-zA-Z0-9_.~<>()|\\[\\]\",@/ –-]+$")
```

Comment:
Backtick is another option

Reply:
Thanks! Added at https://github.com/GoogleCloudPlatform/testgrid/pull/932